governance.md

```diff
@@ -1,133 +1,233 @@
+* Have read the [Contributor Guide](https://spinnaker.io/community/contributing/).
+* Have made multiple contributions to the project or community. Contributions may include, but is not limited to:
+  * Authoring or reviewing PRs on GitHub
+  * Filing or commenting on Issues in GitHub
+  * Contributing to SIG or community discussions (e.g. meetings, Slack, email discussions)
+* Subscribed to [spinnaker-dev@googlegroups.com](TODO)
+* Sponsored by 2 Reviewers. **Note the following requirements for sponsors**:
+  * Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating issues, etc.
+  * Sponsors must be Reviewers or Approvers in any project in the [Spinnaker org](https://github.com/spinnaker).
+  * Sponsors must be from multiple member companies to demonstrate integration across the community.
+* **[Open an issue](https://github.com/spinnaker/governance/issues/new) against the spinnaker/governance repo**
+  * Ensure your sponsors are @mentioned on the issue.
+  * Complete every item on the checklist.
+  * Make sure that the list of contributions included are representative of your work on the project.
+* Have your sponsoring reviewers reply confirmation of sponsorship: `:+1:`.
+* Once your sponsors have responded, your request will be reviewed by the TOC. Any missing information will be requested.
+
+#### Responsibilities and Privileges
+
+* Responsive to issues and PRs assigned to them.
+* Responsive to mentions of SIG teams they are members of.
+* Active owner of code they have contributed (unless ownership is explicitly transferred).
+  * Code is well tested.
+  * Tests consistently pass.
+  * Addresses bugs or issues discovered after code is accepted.
+* They can be assigned to issues and PRs, and people can ask members for reviews.
+
+**Note**: Members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a Reviewer for the project(s) they are active in.
+
+
+### Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of the project. They are knowledgable about both the codebase and software engineering principles. Reviewership is required before becoming an Approver (except under specific conditions; see Approver).
+
+**Defined by**: _reviewers_ entry in the [membership.yml](membership.yml) file.
 
-Duties
+Reviewer status is scoped to a part of the codebase.
 
-* Review, provide meaningful feedback and LGTM PRs
-* Triage GitHub issues, community forum submissions
-* Provide general support in Slack
+#### Requirements
 
-Qualification guidelines
+* An active member of the community.
+* Reviewed at least 5 [substantial](committee-technical-oversight/substantial-contributions.md) PRs to the codebase.
+* Knowledgable about the codebase (exhibited via discussion, issues, thoughtful PR review, etc.)
+* Sponsored by 1 Approver of the project
+  * With no objections from other Approvers
```

Comment:
Sure, we can remove vetos, but I felt like it would be worth having if we're looking to get rid of the TOC voting on each membership / promotion. Let the community do things, and if someone has a problem with an Approver / Reviewer promotion, they'd have the ability to affect it.

Reply:
I don't have a strong opinion one way or another, but I guess one question with vetos is how long you leave the request open...if someone gets a sponsor and opens the PR we'll probably merge as soon as it meets the requirements.  Should we leave these open for a while to wait for objections?  Or remove the status if someone objects within a certain amount of time.  Maybe I'm being pedantic, just wanted to think through what this would look like.